vmdb/spec/models/vm_microsoft_spec.rb

```diff
@@ -0,0 +1,10 @@
+require "spec_helper"
+
+describe VmOrTemplate do
+  context "#supports_migrate?" do
```

Comment:
Actually, this, like the vm_redhat_spec, should only test the overridden methods, so don't test the class methods here, please only do the instance method tests, thanks!